JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -139,6 +139,14 @@ RotationalSpeed,Class,OperatingParameter,IS-A,,,,The rotational speed of a devic
 OperatingProperty,Class,,,,,,A property of an operating parameter of a device (e.g. range of an operating temperature).,https://www.theworldavatar.com/kg/ontodevice/,Operating property
 OperatingRange,Class,OperatingProperty,IS-A,,,,The range for an operating parameter of a device (e.g. range of an operating temperature).,https://www.theworldavatar.com/kg/ontodevice/,Operating range
 OperatingTemperatureRange,Class,OperatingRange,IS-A,,,,The operating temperature range of a device.,https://www.theworldavatar.com/kg/ontodevice/,Operating temperature range
+Activity,Class,,,,,,An event that occurs over a period of time and involves specific actions such as walking or running.,https://www.theworldavatar.com/kg/ontodevice/,Activity
+ActivityType,Class,,,,,,Represents the type of activity being performed (e.g., walking, biking) defined by Google's activity recognition API.,https://www.theworldavatar.com/kg/ontodevice/,Activity Type
+ConfidenceLevel,Class,,,,,,Represents the confidence level indicating the likelihood of the activity being accurate.,https://www.theworldavatar.com/kg/ontodevice/,Confidence Level
+hasActivityType,Object Property,,Activity,ActivityType,,only,Links an activity to its corresponding type (e.g., walking, running).,https://www.theworldavatar.com/kg/ontodevice/,has activity type
+hasConfidenceLevel,Object Property,,Activity,ConfidenceLevel,,only,Links an activity to its confidence level.,https://www.theworldavatar.com/kg/ontodevice/,has confidence level
+prov:wasAssociatedWith,Object Property,,prov:Activity,OntoDevice:Smartphone,,only,Relates an activity with the device that detected it.,https://www.w3.org/ns/prov#,was associated with
```

Comment:
prefixes such as prov: and OntoDevice: are not supported in our converters. What you can do instead is replace prov:wasAssociatedWith with https://www.w3.org/ns/prov#wasAssociatedWith

For OntoDevice:Smartphone, you can simply reference it as Smartphone as the converter will automatically include the prefix https://www.theworldavatar.com/kg/ontodevice/ during the conversion to owl.